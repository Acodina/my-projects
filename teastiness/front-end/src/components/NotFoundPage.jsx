import React from 'react';
import { Link } from 'react-router-dom';
import './NotFoundPage.scss';

function NotFoundPage() {
  return (
    <section className="not-found">
      <h1>Error 404</h1>
      <p>La pagina que est√°s busando no existe.</p>
      <Link to="/">Vuelve a la pagina principal</Link>
    </section>
  );
}

export default NotFoundPage;
