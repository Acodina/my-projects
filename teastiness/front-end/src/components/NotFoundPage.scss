.not-found {

    h1,
    p,
    a {

        margin: 15px;
    }
}