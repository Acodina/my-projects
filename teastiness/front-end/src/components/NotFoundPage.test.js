import renderer from 'react-test-renderer';
import React from 'react';
import NotFoundPage from './NotFoundPage';
import { BrowserRouter as Router } from 'react-router-dom';
describe('App Loading snapshot', () => {
  const tree = renderer.create(
    <Router>
      <NotFoundPage />
    </Router>
  );
  it('should match', () => {
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
