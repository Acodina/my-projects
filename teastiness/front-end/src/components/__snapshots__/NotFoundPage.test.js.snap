// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Loading snapshot should match 1`] = `
<section
  className="not-found"
>
  <h1>
    Error 404
  </h1>
  <p>
    La pagina que est√°s busando no existe.
  </p>
  <a
    href="/"
    onClick={[Function]}
  >
    Vuelve a la pagina principal
  </a>
</section>
`;
